### PR TITLE
Increase code coverage for adapter-rest-impl

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

File: xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/config/IngOptionalParametersFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            .thenThrow(new OAuthException(ResponseHeaders.emptyResponseHeaders(), new ErrorResponse(), ""));
+
+        mockMvc.perform(post(ConsentController.CONSENTS)
+            .contentType(APPLICATION_JSON_VALUE)
+            .content("{}"))
+            .andExpect(status().is(HttpStatus.FORBIDDEN.value()));
+
+        RequestParams params = paramsCaptor.getValue();
+
+        assertThat(params.toMap().containsKey("balanceTypes")).isTrue();
```
Comment (artemsnisarenko) [inline xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/config/IngOptionalParametersFilterTest.java:62]: violates https://rules.sonarsource.com/java/RSPEC-5838

File: xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/controller/ConsentControllerTest.java [hunk 2]
```diff
@@ -52,30 +61,33 @@
...
+
     @BeforeEach
     void setUp() {
         MockitoAnnotations.initMocks(this);
 
         mockMvc = MockMvcBuilders.standaloneSetup(controller)
-            .setMessageConverters(new MappingJackson2HttpMessageConverter())
-            .setControllerAdvice(new RestExceptionHandler(new HeadersMapper()))
-            .build();
+                          .setMessageConverters(new MappingJackson2HttpMessageConverter())
```
Comment (artemsnisarenko) [inline xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/controller/ConsentControllerTest.java:72]: Is the Editor/Code Style/Java/Scheme set to Default in your preferences? Indentation here and on line 83 doesn't look right.

File: xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/config/IngOptionalParametersFilterTest.java (hunk 1 above)
Comment (berezkin88) [reply xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/config/IngOptionalParametersFilterTest.java]: good spot, amended, thx

File: xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/controller/ConsentControllerTest.java (hunk 2 above)
Comment (berezkin88) [reply xs2a-adapter-rest-impl/src/test/java/de/adorsys/xs2a/adapter/controller/ConsentControllerTest.java:72]: The preferences should be fine, I must have added extra tab there. Fixed

**Pull request merged**